### PR TITLE
Documentation: Document potential conflict of ENI with DHCP agents

### LINKED ISSUE #9446: ENI: Node to pod connectivity issue
**General Information**

- Cilium version: 1.6.2
- Orchestration system: self-managed k8s 1.13.2 on AWS EC2 instance
- Kernel 4.16.2, Centos 7.4.1708 with service NetworkManager started
- Configurations:
```
ipam: eni
identity-allocation-mode: crd
blacklist-conflicting-routes: false
enable-endpoint-routes: true
auto-create-cilium-node-resource: true
```
- Cluster description:
```
Two k8s nodes with first ENI(eth0) on AWS subnet1 10.0.1.0/24
pods are on subnet2 192.168.1.0/24
pod ENIs (eth1) are attached to both nodes by cilium-operator

node1
eth0 10.0.1.2
eth1 192.168.1.2

node2
eth0 10.0.1.3
eth1 192.168.1.3
```

**How to reproduce the issue**

1. Create several pods on both nodes
2. Make sure the primary IP address of eth1 on node1 (192.168.1.2) is allocated to a pod
3. From node1 host namespace, ping IP address of a pod on node2 (e.g. 192.168.1.10) , failed

### PR DESCRIPTION
**Pull request by tgraf**
Fixes: #9446

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9934)

### CODE REVIEW

**Comment by tgraf** [comment]
test-docs-please

**Review by aanm — APPROVED** [review]

File: Documentation/gettingstarted/aws-eni.rst [hunk 1]
```diff
@@ -96,3 +95,17 @@ Limitations
...
   like it is coming from outside of the cluster to the receiving pod.
+
+Troubleshooting
+===============
+
+Make sure to disable DHCP on ENIs
+---------------------------------
+
+Cilium will use both the primary and secondary IP addresses assigned to ENIs.
+Use of the primary IP address optimizes the number IPs available to pods but
```
Comment (joestringer) [inline Documentation/gettingstarted/aws-eni.rst]: "number [of] IPs available..."

**Review by joestringer — CHANGES_REQUESTED** [review]
Minor nit, otherwise LGTM.

**Comment by tgraf** [comment]
test-docs-please

**Review by joestringer — APPROVED** [review]

**Pull request merged**